fuzz_testing/test/acl_auto_configure_test.cpp

```diff
@@ -0,0 +1,1752 @@
+// Copyright (C) 2011-2021 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable : 4100) // unreferenced formal parameter
+#endif
+#include <CppUTest/TestHarness.h>
+#ifdef _MSC_VER
+#pragma warning(pop)
+#endif
+
+#include <iostream>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+
+#include <acl_auto.h>
+#include <acl_auto_configure.h>
+#include <acl_auto_configure_version.h>
+#include <acl_thread.h>
+#include <acl_version.h>
+
+#include "acl_test.h"
+
+#include "../fuzz_src/fuzz_testing.h"
+#include <vector>
+
+TEST_GROUP(auto_configure) {
+public:
+  void setup() { preload_data("acl_auto_configure_test"); }
```

Comment:
Can you please add some comments/documentation around the code that you added specially for the fuzz testing so it makes the reader easier to see what code is original and what is not?

Reply:
I have added more documentation at the top of the acl_auto_configure_test.cpp